cmd/ocm/create/cluster/cmd.go

```diff
@@ -511,8 +525,19 @@ func run(cmd *cobra.Command, argv []string) error {
 	return nil
 }
 
-func shouldPromptForBYOVPC(areSubnetsProvided bool) bool {
-	return !args.byovpc.Enabled && args.ccs.Enabled && !areSubnetsProvided && args.interactive
+func shouldPromptForExistingVPC(areSubnetsProvided bool) bool {
+	return !args.existingVPC.Enabled && args.ccs.Enabled && !areSubnetsProvided && args.interactive
+}
+
+func wasClusterWideProxyReceived() bool {
```

Comment:
we can just change it to  clusterWideProxyReceived

Reply:
I rather leave it as is, since the function returns boolean. The function answers the question - "was cluster-wide proxy received"